examples/aerial_authz.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2021 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+import argparse
+from datetime import datetime, timedelta
+
+from google.protobuf import any_pb2, timestamp_pb2
+
+from cosmpy.aerial.client import LedgerClient, NetworkConfig
+from cosmpy.aerial.client.utils import prepare_and_broadcast_basic_transaction
+from cosmpy.aerial.tx import Transaction
+from cosmpy.aerial.wallet import LocalWallet
+from cosmpy.crypto.keypairs import PrivateKey
+from cosmpy.protos.cosmos.authz.v1beta1.authz_pb2 import GenericAuthorization, Grant
+from cosmpy.protos.cosmos.authz.v1beta1.tx_pb2 import MsgGrant
+
+
+def _parse_commandline():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "authz_address",
+        help="address that will be granted authorization to send tokens from wallet",
+    )
+    parser.add_argument(
+        "total_authz_time",
+        type=int,
+        help="authorization time for authz_address in minutes",
+    )
```

Comment:
Suggest making this an optional argument with some default value:
```suggestion
    parser.add_argument(
        "total_authz_time",
        type=int,
        nargs="?",
        default=10,
        help="authorization time for authz_address in minutes",
    )
```